DigitalOffice/Services/UserService/Models/UserCredentials.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace UserService.Models
+{
+    public class UserCredentials
```

Comment:
Hm. UserCredentials? Not UserCredentialsRequest, example?

Reply:
renamed